BoxSelector/index.jsx

```diff
@@ -55,4 +61,6 @@ BoxSelector.propTypes = {
   layout: PropTypes.oneOf(['auto', 'horizontal', 'vertical'])
 }
 
-export default BoxSelector
```

Comment:
The way I understand it `compose(x(A))(B)` is the same as `x(A, B)`. So unless you need more functions `compose` is redundant here.